includes/admin/emails/class-email-notification-util.php

```diff
@@ -285,6 +285,12 @@ public static function get_value( Give_Email_Notification $email, $option_name,
 			? str_replace( '_give_', '', $option_name )
 			: $option_name );
 		$option_value       = give_get_option( $global_option_name, $default );
+		
+		//Emails sent to donors cannot be translated using WPML.
+		//Regiserting only the emails sent to the donors.
+		if (defined('ICL_SITEPRESS_VERSION') && !is_array($option_value)) {
```

Comment:
@ravinderk how come this is not being auto-formatted? Notice the incorrect spacing.

Reply:
@DevinWalker Maybe @amElnagdy did not set up the development environment correctly or GitHub hook did not install for `husky`. We can automate this process on Github by using GitHub action: https://github.com/rtCamp/action-phpcs-code-review